api/src/migrations/extract-user-settings-roles.js

```diff
@@ -51,19 +43,9 @@ module.exports = {
   created: new Date(2016, 4, 26, 15, 10, 0, 0),
   run: () => {
     return getAdmins().then(admins => {
-      promisify(function(callback) {
-        db._users.list({ include_docs: true }, function(err, result) {
-          if (err) {
-            return callback(err);
-          }
-          async.eachSeries(
-            filterResults(result.rows),
-            function(row, callback) {
-              updateUser(admins, row, callback);
-            },
-            callback
-          );
-        });
+      return db.users.allDocs({ include_docs: true }).then(result => {
+        const userRows = result.rows.filter(row => row.id.indexOf('org.couchdb.user:') === 0);
```

Comment:
NB: this could be `.startsWith(..`: http://127.0.0.1:53727/Dash/nvsmoutr/developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/startsWith.html